source/form.html

```diff
@@ -3,134 +3,127 @@
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Пинк</title>
+    <link href="css/style.css" rel="stylesheet">
   </head>
 <body>
-  <div class="header__inner inner">
-    <header class="header">
-      <div class="header__logo-wraper">
-        <a class="page-header__logo">
-          <img src="https://www.explorejasper.com/wp-content/uploads/2019/03/placeholder-images.png" width="74"
-            height="23" alt="Логотип Пинк">
-        </a>
-        <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
-      </div>
-      <nav class="main-nav">
-        <ul class="main-nav__list site-list">
-          <li class="site-list__item">
-            <a class="site-list__link" href="index.html">
-              Главная
-            </a>
-          </li>
-          <li class="site-list__item">
-            <a class="site-list__link" href="photo.html">
-              Фотографии
-            </a>
-          </li>
-          <li class="site-list__item">
-            <a class="site-list__link">
-              Конкурс
-            </a>
-          </li>
-          <li class="site-list__item">
-            <a class="site-list__link" href="https://htmlacademy.ru/study">
-              HTML Academy
-            </a>
-          </li>
-        </ul>
-      </nav>
-  </div>
+  <header class="header">
+    <div class="header__inner inner">
+    <a class="header__logo">
+      <img src="https://www.explorejasper.com/wp-content/uploads/2019/03/placeholder-images.png" width="74" height="23" alt="Логотип Пинк">
+    </a>
+    <button class="header__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+    </div>
+    <nav class="header__main-nav main-nav">
+      <ul class="main-nav__list">
+        <li class="main-nav__item">
+          <a class="main-nav__link" href="index.html">Главная</a>
+        </li>
+        <li class="main-nav__item">
+          <a class="main-nav__link" href="photo.html">Фотографии</a>
+        </li>
+        <li class="main-nav__item">
+          <a class="main-nav__link">Конкурс</a>
+        </li>
+        <li class="main-nav__item">
+          <a class="main-nav__link" href="https://htmlacademy.ru/study">HTML Academy</a>
+        </li>
+      </ul>
+    </nav>
+</header>
 
 <main>
 
-<section class="app">
-  <div class="app__inner inner"  >
-    <h1>Конкурс на самое безбашенное путешествие с Пинк</h1>
+<section class="competition">
+  <div class="competition__inner inner">
+    <h1 class="competition-slogan">Конкурс на самое безбашенное путешествие с Пинк</h1>
```

Comment:
```suggestion
    <h1 class="competition__slogan">Конкурс на самое безбашенное путешествие с Пинк</h1>
```

иначе с тире это совсем другой блок-класс, и совсем другой файл стилей